Procfile

```diff
@@ -0,0 +1 @@
+web: bin/rails server -p $PORT -e $RAILS_ENV
```

Comment:
Minor, the commit message for this is spelt wrong s/Profile/Procfile/

Reply:
Thanks - fixed